assets/html/home.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="../css/home.css">
+    <!-- <meta name="viewport" content="width=device-width,initial-scale=1"> -->
```

Comment:
Same as above, I'm curious on why this is commented out 🤔

Reply:
For this one, I want add slight changes after the main style has fixed

Reply:
so now it is commented out and then applied later

Reply:
ok sounds good to me

Usually I will leave a "TODO: blahblahblah" comment to remind myself to come back and finish what I started